docs/user-guide/sdks-using.md

```diff
@@ -64,6 +70,25 @@ Pull the packages from an online registry such as npm or PyPi.
     - To define the versioning scheme for Node packages, use [semantic versioning](https://docs.npmjs.com/about-semantic-versioning).
 
     - To define versioning for Python packages, specify versions or version ranges in a `requirements.txt` file checked-in to your project. For more information, see [pip install](https://pip.pypa.io/en/stable/cli/pip_install/) in the pip documentation.
+### Install Java SDK from online registry
```

Comment:
Curious if we can combine the 3 into a single header?

If we feel that the steps might be too distinct, I think we should just split them all into their own sub-header (one for node, one for java, and one for python - in that order) 😋

Reply:
I think the java step is too distinct. We can spilt them each instead as you noted. I will split them in order you noted and check it in.

Reply:
I made a new check in with them split and reorder.